app/main.py

```diff
@@ -16,4 +16,7 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {"mutable": [my_favourite_films, marks,
```

Comment:
format your code in according next structure
<img width="314" alt="Screenshot 2023-10-11 at 16 37 56" src="https://github.com/mate-academy/py-mutable-immutable/assets/117740672/ecc488be-5900-4378-845a-308933f265f9">